packages/nuxt/src/components/loader.ts

```diff
@@ -58,7 +58,7 @@ export const loaderPlugin = createUnplugin((options: LoaderOptions) => {
             !components.some(c => c.pascalName === component.pascalName && c.mode === 'client')
           if (isServerOnly) {
             imports.add(genImport(serverComponentRuntime, [{ name: 'createServerComponent' }]))
-            imports.add(`const ${identifier} = createServerComponent(${JSON.stringify(name)})`)
+            imports.add(`const ${identifier} = createServerComponent(${JSON.stringify(pascalCase(name))})`)
```

Comment:
Does this also work?

```suggestion
            imports.add(`const ${identifier} = createServerComponent(${JSON.stringify(component.pascalName)})`)
```

Reply:
yes !